modules/javafx.swing/src/main/java/javafx/embed/swing/package.html

```diff
@@ -37,11 +37,24 @@
 
 <body>
 <p>Provides the set of classes to use JavaFX inside Swing applications.</p>
-<p>The {@link javafx.embed.swing.JFXPanel JFXPanel} class is the base class
-that provides JavaFX and Swing interoperability. This class defines a
-lightweight Swing component, which accepts and renders an instance of
-{@link javafx.scene.Scene Scene} and forwards all input events from Swing
-to the attached JavaFX scene.</p>
+<p>Defines APIs for the JavaFX / Swing interop support included with the
+   JavaFX UI toolkit, including {@link javafx.embed.swing.SwingNode} (for
+   embedding Swing inside a JavaFX application) and
+   {@link javafx.embed.swing.JFXPanel} (for embedding JavaFX inside a Swing
+   application). </p>
+<p>The {@link javafx.embed.swing.JFXPanel JFXPanel} class defines a
+   lightweight Swing component, which accepts and renders an instance of
+   {@link javafx.scene.Scene Scene} and forwards all input events from Swing
+   to the attached JavaFX scene.
+   The {@link javafx.embed.swing.SwingNode} class is used to embed
+   a Swing content into a JavaFX application.
+   The content to be displayed is specified with the SwingNode.setContent method
+   that accepts an instance of Swing {@code JComponent}. The hierarchy of components
+   contained in the {@code JComponent} instance should not contain any heavyweight
+   components, otherwise {@code SwingNode} may fail to paint it. The content gets
+   repainted automatically. All the input and focus events are forwarded to the
+   {@code JComponent} instance transparently to the developer.</p>
```

Comment:
I might suggest removing the "transparently to the developer" part of this from the package description, since you don't also say that for `JFXPanel`, and someone might wonder whether there is something they need to do in the `JFXPanel` case. Both of the class descriptions include this language, so I think that's sufficient.